tests/ttnn/unit_tests/operations/test_moreh_softmax.py

```diff
@@ -471,3 +471,72 @@ def test_softmax_backward_optional_output_tensor(shape_dim, optional_output_tens
     passing, out = comp_allclose_and_pcc(x.grad, tt_dev, rtol=rtol, atol=atol)
     logger.info(out)
     assert passing
+
+
+@pytest.mark.parametrize(
+    "shape_dim_strategy",
+    [
+        [[32, 32], 1, ttnn.operations.moreh.SoftmaxOpParallelizationStrategy.SMALL_W],
+        [[32, 32], 0, ttnn.operations.moreh.SoftmaxOpParallelizationStrategy.SMALL_H],
+        [[32, 32], 1, ttnn.operations.moreh.SoftmaxOpParallelizationStrategy.LARGE_W],
+        [[32, 32], 0, ttnn.operations.moreh.SoftmaxOpParallelizationStrategy.LARGE_H],
+        [[1, 1, 32, 32], 1, ttnn.operations.moreh.SoftmaxOpParallelizationStrategy.LARGE_C],
+        [[1, 1, 32, 32], 0, ttnn.operations.moreh.SoftmaxOpParallelizationStrategy.LARGE_C],
+    ],
+)
+def test_softmax_bfp8(shape_dim_strategy, device):
+    # TODO @thanhnguyen-moreh: Support bfloat8_b in kernel
+    pytest.skip(f"bfloat8_b is not supported in the kernel")
+    shape, dim, strategy = shape_dim_strategy
+    torch.manual_seed(0)
+
+    x = torch.randint(low=0, high=4, size=shape).to(torch.bfloat16)
```

Comment:
Should be random float value initialized.

Reply:
Resolved